tests/network/setup_multimachine.pm

```diff
@@ -24,7 +24,11 @@ sub run {
     my ($self) = @_;
     my $hostname = get_var('HOSTNAME');
     select_console 'root-console';
-    my $nm_id = is_sle('15-sp4+') ? 'eth0' : 'Wired connection 1';
+    # delimiter is 2 spaces, because name can contain space, between columns are at least 2 spaces
+    if (is_networkmanager) {
+        my $nm_id = script_output(q(nmcli connection show|awk -F'  ' '/ethernet/ {print$1}'));
+        my $device = script_output(q(nmcli connection show|awk -F '  ' '/ethernet/ {print$4}'));
```

Comment:
At least this one does not work on my connection name, which does happen to have a space in the name

```
> nmcli connection show|awk -F "  "  '/ethernet/ {print $4}'

```

what DOES work though is:

```

> nmcli connection show| awk  '/ethernet/ {print $NF}'
enp0s25
```

(NF being number of fields - and in this case we care for the last field, so this makes things easy)

Reply:
This works, it can be done differently when it does not work.
Problem is not `device` but `name` on SLED.
![image](https://user-images.githubusercontent.com/9447312/151371550-9232ffda-be48-4c65-8579-552cf4877f2f.png)

Reply:
I will use arguments Fabian mentioned.